### DISCUSSION TITLE
Demo of how to run caddy with socket activation + rootless Podman + Quadlet files

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#24199] [Show and tell]
Caddy 2.9.0 (not yet released) includes support for _socket activation_.
I tried it out. The functionality works fine with Podman.

The cool thing is that `X-Forwarded-For` is now supported when running caddy with rootless podman (with pasta).
Caddy can act as a reverse proxy for a custom network (i.e. a network that was created with `podman network create mynet` or with a quadlet).

I wrote some examples here:

https://github.com/eriksjolund/podman-caddy-socket-activation

**Comment by ygalblum** [comment]
Great examples. Would you like to maybe add a link to your repo in the README of the https://github.com/containers/appstore repo?

**Reply by eriksjolund** [reply]
Sure, I opened a PR https://github.com/containers/appstore/pull/25
